### DISCUSSION TITLE
What an introduction about pharmacophores should have?

### DISCUSSION THREAD

**Discussion by dprada** [uibcdf/Study_Group_Pharmacophores#15] [Ideas]
In case an introduction paragraph has to be written in a review of pharmacophores: what would be the papers to be cited? and the concepts, arguments, and topics to be mentioned?

**Comment by dprada** [comment]
## Similar structure and chemical attributes will have similar experimental attributes and activities

This concept is a belief on which screening with ligand-based pharmacophores assumes. But this ansatz is not always valid. In #3 there is a discussion about the validity and shortcomings of this assumption.

**Reply by dprada** [reply]
In #3 there are some references of works answering the question: How valid is the idea that chemical similarity entails similar biological activity?

[Muchmore, Steven W, Derek A Debe, James T Metz, Scott P Brown, Yvonne C Martin, and Philip J Hajduk. “Application of Belief Theory to Similarity Data Fusion for Use in Analog Searching and Lead Hopping.” *Journal of Chemical Information and Modeling* 48, no. 5 (2008): 941–48.](https://doi.org/10.1021/ci7004498)

[Yan, Xin, Chenzhong Liao, Zhihong Liu, Arnold T Hagler, Qiong Gu, and Jun Xu. “Chemical Structure Similarity Search for Ligand-Based Virtual Screening: Methods and Computational Resources.” *Current Drug Targets* 17, no. 14 (2016): 1580–85.]( https://doi.org/10.2174/1389450116666151102095555.)

[Muegge, Ingo, and Prasenjit Mukherjee. “An Overview of Molecular Fingerprint Similarity Search in Virtual Screening.” *Expert Opinion on Drug Discovery* 11, no. 2 (2015): 137–48.](https://doi.org/10.1517/17460441.2016.1117070)

[Eckert, Hanna, and Jürgen Bajorath. “Molecular Similarity Analysis in Virtual Screening: Foundations, Limitations and Novel Approaches.” Drug Discovery Today 12, no. 5–6 (2007): 225–33.](https://doi.org/10.1016/j.drudis.2007.01.011)

**Comment by dprada** [comment]
## Introduction to pharmacophores

In #3, Zoe Cournia et alt. make use of the following reference #4 to introduce the pharmacophores approaches for the first time in their paper:

[Yang, Sheng-Yong. “Pharmacophore Modeling and Applications in Drug Discovery: Challenges and Recent Advances.” Drug Discovery Today 15, no. 11–12 (2010): 444–50](https://doi.org/10.1016/j.drudis.2010.03.013)

**Reply by Daniel-Ibarrola** [reply]
Also, #5 gives a good introduction to pharmacophores including ligand and structure based approaches. Likewise, #10 does a jood gob introducing the concept.

**Reply by dprada** [reply]
And we can not forget "the seminal" paper by Ehrlich in 1909 and paper #9 "setting the record straight":

[Ehrlich, P. “Über Den Jetzigen Stand Der Chemotherapie.” Berichte Der Deutschen Chemischen Gesellschaft 42, no. 1 (2013): 17–47.](https://doi.org/10.1002/cber.19090420105)

**Reply by dprada** [reply]
About the history and evolution of the pharmacophore concept:

[Guner, Osman. “History and Evolution of the Pharmacophore Concept in Computer-Aided Drug Design.” Current Topics in Medicinal Chemistry 2, no. 12 (2002): 1321–32.](https://doi.org/10.2174/1568026023392940)

[Wermuth, Camille G. *Pharmacophores: Historical Perspective and Viewpoint from a Medicinal Chemist*. Edited by Thierry Langer and Rémy D. Hoffmann. Methods and Principles in Medicinal Chemistry. Wiley Online Library, 2006.](https://doi.org/10.1002/3527609164.ch1)

**Reply by Daniel-Ibarrola** [reply]
The thesis by Tunca (#24 ) also has a good introduction to computer aided drug design. It covers pharmacophores, docking and molecular dynamics.

**Reply by Daniel-Ibarrola** [reply]
#42 can be useful for writing an introduction.
